packages/ts/form/test/Field.test.ts

```diff
@@ -739,6 +740,52 @@ describe('form/Field', () => {
       });
     });
 
+    [
+      { model: binder.model.fieldArrayString as AbstractModel<any>, value: ['a', 'b'] },
+      { model: binder.model.fieldArrayModel as AbstractModel<any>, value: [{ idString: 'id' }] },
+    ].forEach(async ({ model, value }) => {
```

Comment:
This approach is unsupported by Idea (I cannot run test created this way using Idea's interface), so it would be better to use two tests that runs the generalized function.

Reply:
Actually there are many other tests written like this one in the same file and maybe somewhere else. We should change them all if that's a problem.